gateway/events_map.go

```diff
@@ -73,6 +73,4 @@ var EventCreator = map[string]func() Event{
 
 	"RELATIONSHIP_ADD":    func() Event { return new(RelationshipAddEvent) },
 	"RELATIONSHIP_REMOVE": func() Event { return new(RelationshipRemoveEvent) },
-
-	"APPLICATION_COMMAND_UPDATE": func() Event { return new(ApplicationCommandUpdateEvent) },
```

Comment:
I'm not aware of anything that signifies the removal of this.

Reply:
I'm giving this a go anyway. It doesn't seem to be part of the documented API.

Reply:
https://github.com/discord/discord-api-docs/pull/3691

Reply:
Yeah it used to be in the documented API but apparently wasn't meant to be there ??? and was removed afterwards.